ClientSide(FrontEnd)/clay_/src/components/login.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
use one import statement for react
import React, { useState } from 'react';

Reply:
i removed extra import